packages/frontend-v2/utils/theme.ts

```diff
@@ -1,4 +1,5 @@
 import { extendTheme } from "@chakra-ui/react";
+import { Button } from "./component_styles/buttons"
```

Comment:
You were right, the theme file is actually getting quite cluttered now. We can break it up and make theme its own directory under utils. Checkout out the [scaling out design section in the docs](https://chakra-ui.com/docs/styled-system/theming/customize-theme#scaling-out-your-project)!

According to the above example, we can move the components/Button into the new theme directory, and the current theme file will be theme/index.ts.